### ISSUE TITLE
UnnecessaryFullyQualifiedName: false positive on property named 'kotlin'

### ISSUE THREAD

**Issue by eygraber** [detekt/detekt#9282]
## Expected Behavior
A function call on a property named `kotlin` doesn't violate `UnnecessaryFullyQualifiedName`

## Observed Behavior
It does

## Steps to Reproduce

[Declaration](https://github.com/eygraber/gradle-conventions/blob/master/conventions-plugin/src/main/kotlin/com/eygraber/conventions/GradleConventionsPluginExtension.kt#L198):

```kotlin
internal val kotlin = GradleConventionsKotlin()
    get() {
      wasKotlinUserConfigured = true

      return field
    }
```

[Usage](https://github.com/eygraber/gradle-conventions/blob/master/conventions-plugin/src/main/kotlin/com/eygraber/conventions/GradleConventionsPluginExtension.kt#L78):

```kotlin
@Suppress("UnnecessaryFullyQualifiedName")
kotlin.configure(isKotlinConfigured)
```
## Your Environment

* Version of detekt used: 2.0.0-alpha.3
* Link to your project (if it's a public repository): linked in examples above
